### PR TITLE
LOG-4464: Document keys used for secrets

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:

* Documents the opinionated keys that are recognized for ClusterLogForwarder outputs

### Links
* https://issues.redhat.com/browse/LOG-4464

### CODE REVIEW

**Comment by jcantrill** [comment]
image

**Comment by jcantrill** [comment]
cc @libander

File: apis/logging/v1/cluster_log_forwarder_types.go [hunk 1]
```diff
@@ -383,3 +383,48 @@ type LimitSpec struct {
...
+	//An AWS access key ID
+	AWSAccessKeyID string `json:"aws_access_key_id"`
+	//An AWS credentials key
+	AWSCredentialsKey string `json:"credentials"`
+	//An AWS role ARN
+	AWSWebIdentityRoleKey string `json:"role_arn"`
+	//The HEC token for authorizing against a Splunk endpoint
+	SplunkHecToken string `json:"hecToken"`
+	//The Google application credentials JSON
+	GoogleeAppCredentials string `json:"google-application-credentials.json"`
```
Comment (cahartma) [inline apis/logging/v1/cluster_log_forwarder_types.go:429]: Spelling

**Review by cahartma — COMMENTED** [review]
One minor spelling, then good to me.

File: docs/reference/operator/api.adoc [hunk 2]
```diff
@@ -538,10 +539,24 @@ CollectorSpec is spec to define scheduling and resources for a collector
...
+===== Description
+
+=====  Type
+* array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|name|string|  Name must match the name of one entry in pod.spec.resourceClaims of
```
Comment (cahartma) [inline docs/reference/operator/api.adoc:557]: Should be: 
`Name must match the name of one entry in pod.spec.resourceClaims of the Pod where this field is used. It makes that resource available inside a container.`

Comment (cahartma) [reply docs/reference/operator/api.adoc:557]: ... or just remove the trailing "of" if you intended to shorten

**Pull request merged**